<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1601</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>compiler like C</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>10</y>
      <width>471</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Adobe Devanagari&quot;;</string>
    </property>
    <property name="text">
     <string>compiler like C</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="lexerBrowser">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>391</y>
      <width>331</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="ParserBrowser">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>100</y>
      <width>361</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="lexButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>670</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>lexical analysis</string>
    </property>
   </widget>
   <widget class="QPushButton" name="synButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>670</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>syntax and sematic analysis</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>331</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelInput">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Adobe Devanagari&quot;;</string>
    </property>
    <property name="text">
     <string>input</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInput_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Adobe Devanagari&quot;;</string>
    </property>
    <property name="text">
     <string>lexicial result</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInput_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Adobe Devanagari&quot;;</string>
    </property>
    <property name="text">
     <string>syntax result</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IRBrowser">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>100</y>
      <width>321</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelInput_4">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>70</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Adobe Devanagari&quot;;</string>
    </property>
    <property name="text">
     <string>intermediate representation</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="CodeBrowser">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>100</y>
      <width>321</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="codeButton">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>670</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>object code generation</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInput_5">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>70</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Adobe Devanagari&quot;;</string>
    </property>
    <property name="text">
     <string>object code</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1601</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="actionOpenFile"/>
   </widget>
   <widget class="QMenu" name="menuhelp">
    <property name="title">
     <string>help</string>
    </property>
    <addaction name="actionHow_to_use"/>
   </widget>
   <addaction name="menufile"/>
   <addaction name="menuhelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionopen_file">
   <property name="text">
    <string>open file</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="text">
    <string> open file to analyze</string>
   </property>
  </action>
  <action name="actionHow_to_use">
   <property name="text">
    <string>How to use?</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
